packages/publications/tests/Feature/PublicationTypeTest.php

```diff
@@ -423,7 +424,8 @@ public function testJsonRepresentationWithDefaultValues()
                 "sortField": "__createdAt",
                 "sortAscending": true,
                 "pageSize": 0,
-                "fields": []
+                "fields": [],
+                "meta": []
```

Comment:
@rgasch what do you think about only storing the meta array when it's not empty? That means it wouldn't be present in defaultly generated files, but could easily be added by the user? 

Pros/cons IMO
+ Cleaner schema
- Extra steps for users

Of course the field will still be documented either case.